src/openloco/environment.cpp

```diff
@@ -158,13 +158,18 @@ namespace openloco::environment
         if (!fs::exists(result))
         {
 #ifndef _WIN32
-            if (auto similarResult = find_similar_file(result); similarResult.empty())
+            auto similarResult = find_similar_file(result);
+            if (similarResult.empty())
             {
-#endif
                 std::cerr << "File not found: " << result << std::endl;
-#ifndef _WIN32
+            }
+            else
+            {
                 result = similarResult;
```

Comment:
You could've just put this in `else` block and not duplicate error reporting and keep changes to minimum. I'm not very fussed about it though, it looks fine otherwise.

Reply:
The original `#ifdefs` made it rather difficult to read

Reply:
Yeah, Marijn and I discussed the patch and thought we'd improve readability while at it...